keras_cv/models/convnext_test.py

```diff
@@ -51,6 +51,12 @@
 ]
 
 
+@pytest.mark.skipif(
+    "INTEGRATION" not in os.environ or os.environ["INTEGRATION"] != "true",
```

Comment:
This annotation is bit long, and is kind of pain to copy/paste to every integration test. Is it possible to consolidate integration tests based on file/directory? so that we can skip them altogether.